<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1241</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(64,224,208);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>RDV</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>741</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_AR">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_MR">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SR">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_RR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_TR">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>280</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>tri</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>212</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font :10pt&quot;NV Boli&quot;;</string>
         </property>
         <property name="text">
          <string>Id patient</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineIDP1">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineIDM">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font :10pt&quot;NV Boli&quot;;</string>
         </property>
         <property name="text">
          <string>Date/Heure</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineDR">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font :10pt&quot;NV Boli&quot;;</string>
         </property>
         <property name="text">
          <string>Sujet</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineSu">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font :10pt&quot;NV Boli&quot;;</string>
         </property>
         <property name="text">
          <string>Id medecin</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>60</y>
        <width>212</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">font :10pt&quot;NV Boli&quot;;</string>
         </property>
         <property name="text">
          <string>Adresse</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ecrire_mail">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="styleSheet">
          <string notr="true">font :10pt&quot;NV Boli&quot;;</string>
         </property>
         <property name="text">
          <string>Objet</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ecrire_objet">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">font :10pt&quot;NV Boli&quot;;</string>
         </property>
         <property name="text">
          <string>Message</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="ecrire_txt"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_mail">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>270</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Facture</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>11</y>
        <width>711</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_AF">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>250</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_MF">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SF">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>170</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRF">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTF">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>280</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>tri</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>248</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font :10pt&quot;NV Boli&quot;;</string>
         </property>
         <property name="text">
          <string>Id patient</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineIDP2">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font :10pt&quot;NV Boli&quot;;</string>
         </property>
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineDF">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font :10pt&quot;NV Boli&quot;;</string>
         </property>
         <property name="text">
          <string>Service</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineSe">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">font :10pt&quot;NV Boli&quot;;</string>
         </property>
         <property name="text">
          <string>Mode paiment</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineMP">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">font :10pt&quot;NV Boli&quot;;</string>
         </property>
         <property name="text">
          <string>Libellé</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineL">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButtonPDF">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QLabel" name="Date">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="Timer">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1275</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
